azuresqlworkshop/01-IntroToAzureSQL.md

```diff
@@ -115,10 +159,10 @@ In this section, you've learned about Azure SQL and the deployment options that
 
 
 If you want to dive deeper into the deployment options and how to choose, check out the following resources:  
-* [Blog announcement for Azure SQL](https://techcommunity.microsoft.com/t5/Azure-SQL-Database/Unified-Azure-SQL-experience/ba-p/815368) which explains and walks through Azure SQL and some of the resulting views and experiences available in the Azure portal.
```

Comment:
For some reason the `.` in this URL was removed? can you add it back and commit?

Reply:
fixed